virttest/utils_misc.py

```diff
@@ -196,6 +196,23 @@ def get_usable_memory_size(align=None):
     return usable_mem
 
 
+def align_value(value_to_be_aligned, align=1024):
+    """
+    Align the 'value_to_be_aligned' to 1024 multiples by default
+
+    :param value_to_be_aligned: value to be checked or changed
+                                to 'align' multiples
+    :param align: value used for align
+    :return: aligned value
+    """
+    value_aligned = int(value_to_be_aligned)
+    align = int(align)
+    if value_aligned % align != 0:
+        # // get the divisor only in int
+        value_aligned = ((value_aligned // align) + 1) * align
+    return value_aligned
```

Comment:
An one line implementation could be:
`return int(math.ceil(float(value) / factor) *  factor)`

Reply:
agree

Reply:
@luckyh Thanks for one liner, I thought of not using any modules, but I am fine with this approach.